Royalty/DefInjected/ResearchProjectDef/Apparel_Various.xml

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<LanguageData>
+  
+  <!-- EN: cataphract armor -->
+  <CataphractArmor.label>катафрактарская броня</CataphractArmor.label>
+  <!-- EN: Craft cataphract armor, heavy powered armor that slows the user but which can absorb extreme punishment. Note that these also require advanced components. -->
+  <CataphractArmor.description>Изготовление тяжелых энергетических доспехов, которые замедляют пользователя, но могут поглотить чрезвычайный урон. Для производства требуются продвинутые компоненты.</CataphractArmor.description>
```

Comment:
```suggestion
  <CataphractArmor.description>Изготовление тяжелых силовых доспехов, которые замедляют носящего, но могут поглотить огромное количество урона. Для производства требуются продвинутые компоненты.</CataphractArmor.description>
```

Reply:
Fixed in ed894251a4494d7065d88e906e1aa3491334fb13